main.py

```diff
@@ -50,15 +47,11 @@
 
     try:
         import pystray
-    except:
```

Comment:
Why are you changing which exceptions are caught? This will likely cause issues eventually as the code changes overtime, especially since this is close to the top-level of the program.